src/binaryen-c.h

```diff
@@ -2124,6 +2124,12 @@ BINARYEN_API BinaryenIndex BinaryenGetOneCallerInlineMaxSize(void);
 // Applies to all modules, globally.
 BINARYEN_API void BinaryenSetOneCallerInlineMaxSize(BinaryenIndex size);
 
+// Gets the currently set of inlining heavyweight functions.
```

Comment:
```suggestion
// Gets whether heavyweight functions are allowed to be inlined.
// Applies to all modules, globally.
```
Does that describe it accurately?